packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -77,6 +83,7 @@ export const defaultCacheExpiryTimeoutMs: number = 2 * 24 * 60 * 60 * 1000; // 2
  * server can match it with its epoch value in order to match the version.
  * It also validates the epoch value received in response of fetch calls. If the epoch does not match,
  * then it also clears all the cached entries for the given container.
+ * @public
  */
 export class EpochTracker implements IPersistedFileCache {
```

Comment:
It would probably be worth double checking if anything extends this class within this package. If not, I would recommend also marking this class as `@sealed`.

Reply:
@Josmithr EpochTrackerWithRedemption extends EpochTracker. Should it just be left as public?

Reply:
Okay. In that case, yes, let's leave it as public and not sealed.